src/20_Components/amp-springboard-player.html

```diff
@@ -0,0 +1,40 @@
+<!-- 
+  #### Introduction
+
+  Embed videos hosted on the Springboard Video Platform in your AMP HTML files.
+-->
+<!-- -->
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <link rel="canonical" href="<%host%>/components/amp-vimeo/">
+  <!-- #### Setup -->
+  <!--
+    Import the `amp-springboard-player` component in the header.
+  -->
+  <script async custom-element="amp-springboard-player" src="https://cdn.ampproject.org/v0/amp-springboard-player-0.1.js"></script>
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+</head>
+<body>
+
+  <!-- #### Basic Usage -->
+  <!--
+    Embed videos hosted on the Springboard Video Platform via the `data-site-id`, `data-domain`, `data-player-id`, `data-content-id` attributes.
+    The `data-mode` attribute is used to determine if this is a single video or playlist.  If it's a playlist, the `data-items` attribute specifies the number of items in the playlist.
```

Comment:
I don't have any example content for a playlist unfortunately.  I'll reach out to springboard for a sample playlist.